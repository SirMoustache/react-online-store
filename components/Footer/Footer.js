/**
 * Absolute imports
 */
import React from 'react';
import styled from 'styled-components';

const Root = styled.footer`
  flex: 0 0 auto;
`;

const Footer = () => <Root>Footer placeholder</Root>;

export default Footer;