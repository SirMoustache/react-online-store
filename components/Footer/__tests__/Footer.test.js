/**
 * Absolute imports
 */
import React from 'react';
import { render, cleanup } from 'react-testing-library';

/**
 * Local Components
 */
import Footer from '..';

const renderComponent = ({ ...rest } = {}) => render(<Footer {...rest} />);

describe('<Footer />', () => {
  afterEach(cleanup);

  it('should match snapshot', () => {
    const { container } = renderComponent();
    expect(container).toMatchSnapshot();
  });

  it('should render an <footer> tag', () => {
    const { container } = renderComponent();
    expect(container.querySelector('footer')).not.toBeNull();
  });
});
