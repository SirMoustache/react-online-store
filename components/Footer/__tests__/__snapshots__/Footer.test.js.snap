// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render correctly 1`] = `
<Footer__Root>
  Footer placeholder
</Footer__Root>
`;
